agent/testlib/mock_resource_server.js

```diff
@@ -336,7 +336,7 @@ function get_config_map(name, type, key, content) {
 }
 
 ConfigMapServer.prototype.add_address_definition = function (def, name, annotations, status) {
-    var address = {kind: 'Address', metadata: {name: def.address}, spec:def};
+    var address = {kind: 'Address', metadata: {name: name}, spec:def};
```

Comment:
I don't think we need this change. If we want it I would do name: name || def.address as the name is not always specified.